iron/distribution.yaml

```diff
@@ -5885,8 +5885,8 @@ repositories:
     release:
       tags:
         release: release/iron/{package}/{version}
-      url: https://github.com/ros2-gbp/septentrio_gnss_driver_ros2-release.git
-      version: 1.2.3-3
+      url: https://github.com/septentrio-users/septentrio_gnss_driver_ros2-release
```

Comment:
Are you sure that you really want to do this?  If you do, then your releases for Rolling will go to the `ros2-gbp` organization, while your releases for Iron will go to the `septentrio-users` organization.  This is allowed, but will probably confuse you later on.

If you really do want to do this, then at the very least this needs to be suffixed with `.git` to pass the linters.